src/dug/core/annotate.py

```diff
@@ -352,10 +352,17 @@ def make_request(self, curie: str, http_session: Session):
         url = f"{self.url}{urllib.parse.quote(curie)}"
 
         try:
-            response = http_session.get(url).json()
-            return response
+            response = http_session.get(url)
+            if response.status_code == 200:
+                return response.json()
+            if response.status_code == 400:
+                logger.error(f"No synonyms returned for: `{curie}`. Validation error.")
+                return []
+            if response.status_code == 500:
+                logger.error(f"No synonyms returned for: `{curie}`. Internal server error from {self.url}.")
+                return []
         except json.decoder.JSONDecodeError as e:
-            logger.error(f"No synonyms returned for: {curie}")
+            logger.error(f"No synonyms returned for: `{curie}`. Json parse error.")
```

Comment:
We should also log either the exception's message or the string that caused the parse error

Reply:
maybe log the url that was used ?